### ISSUE TITLE
your model does not fully load the pre-trained weight.

### ISSUE THREAD

**Issue by FLyingLSJ** [ifzhang/FairMOT#36]
I try model only with MOT15 dataset
![image](https://user-images.githubusercontent.com/18750080/79402038-5af87400-7fbd-11ea-9125-738105d12c13.png)

When I run `sh experiments/all_dla34.sh` I got this info。
![image](https://user-images.githubusercontent.com/18750080/79402090-7d8a8d00-7fbd-11ea-9400-de8920330f1e.png)

Is that normal?

**Comment by ifzhang** [comment]
Yes, it's normal. We just load the backbone parameters of the pretrained model.

**Comment by tfzhou** [comment]
@ifzhang only backbone parameters are loaded when we finetune the models on MOT15 and MOT20? I can nearly reproduce the validation results when I train from scratch, however, after finetuning on MOT15, the validation performance on MOT15 heavily degrades.

**Comment by ifzhang** [comment]
@tfzhou When finetuning on MOT15, you should load the model which you train from scratch, like my baseline model all_dla34.pth. All parameters should be loaded. After finetuning on MOT15, you should set --conf_thres 0.3 or 0.4. I can get 80+ MOTA on the validation set after finetuning on MOT15 for 10 epochs. If still degraded results, you may check whether the MOT15 data is processed correctly.

**Comment by tfzhou** [comment]
@ifzhang Cheers. I obtain a 80+ MOTA score on the validation set after setting the conf_thres to 0.3.
